core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/dr/BackupRestoreSaltConfigGenerator.java

```diff
@@ -0,0 +1,44 @@
+package com.sequenceiq.cloudbreak.reactor.handler.cluster.dr;
+
+import com.sequenceiq.cloudbreak.orchestrator.model.SaltConfig;
+import com.sequenceiq.cloudbreak.orchestrator.model.SaltPillarProperties;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+
+import static java.util.Collections.singletonMap;
+
+@Component
+public class BackupRestoreSaltConfigGenerator {
+
+    public SaltConfig createSaltConfig(String location, String backupId, String cloudPlatform) throws URISyntaxException {
+        String fullLocation = buildFullLocation(location, backupId, cloudPlatform);
+
+        Map<String, SaltPillarProperties> servicePillar = new HashMap<>();
+        servicePillar.put("disaster-recovery", new SaltPillarProperties("/postgresql/disaster_recovery.sls",
+            singletonMap("disaster_recovery", singletonMap("object_storage_url", fullLocation))));
+        return new SaltConfig(servicePillar);
+    }
+
+    public String buildFullLocation(String location, String backupId, String cloudPlatform) throws URISyntaxException {
```

Comment:
this should be either private or in a separate class, as it's not called from anywhere else

Reply:
backupid is not used

Reply:
there is a class, `CloudStorageConfigGenerator` maybe it worth checking out if you could use/reuse some parts of it

Reply:
It's called from `BackupRestoreSaltConfigGeneratorTest`. I've made it package local.
I accidentally deleted the part where backupId was used when I refactored this code. It's been added back.
I looked at `CloudStorageConfigGenerator`.  I prefer what I'm doing here, so unless there's a strong reason to change it I'm going to keep what I have.

Reply:
at least `"aws"` and `"azure"` should be replaced with `CloudConstants`/`CloudPlatform`